### DISCUSSION TITLE
Thoughts from Digital Data 8

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#7820] [Ideas]
# Day 1 - May 29, 2024

## Interesting Resources

 - iDigBio Digitization Academy: https://www.idigbio.org/tags/digitization-academy
 - Digitization Coordination Workshop Report: https://doi.org/10.3897/rio.10.e120626
 - Convention on Biological Diversity: https://www.cbd.int/convention/text
 - TRY plant trait database: https://www.try-db.org/TryWeb/Home.php
 - Open Traits network: https://opentraits.org/
 - Plant Phenology Ontology: https://obofoundry.org/ontology/ppo.html
 - Planteome Reference Ontologies for Plants: https://browser.planteome.org/amigo
 - LeafMachine2  (can extract trait data from at least 245 angiosperm families and calculate pixel-to-metric conversion factors for 26 commonly used ruler types.): https://leafmachine.org/
 - VoucherVision (a multi-institution collaboration with the goal of significantly automating specimen label transcription using Large Language Models.): https://huggingface.co/spaces/phyloforfun/VoucherVision

## Managing Biodiversity Data in the Blockchain: Full Tracking of All Data Transactions Through Time in Managing Systematic Collections - Town Peterson

**Current Biodiversity Data Challenges**
 - Records are dynamic
 - Data exists as a 'single final version' without retention of previous versions
 - No changes tracking (not true for some things in Arctos)
 - No user-based tracking (not true for some things in Arctos)

KU  Students built a prototype of a Blockchain-like tool which was not demoed and isn't ready for prime time.

The audience asked
 - Is Blockchain really necessary? Just use existing tools to track history and data provenance (Arctos does this fairly well.)
 - What happens when the data leaves? If everyone was using Blockchain, no problem (as if...)
 - Use a Merkle-tree data structure instead? (What's that?! - A hash tree, also known as a Merkle tree, is a tree in which each leaf node is labeled with the cryptographic hash of a data block, and each non-leaf node is labeled with the cryptographic hash of its child nodes' labels.)
 
## Hosts, Parasites and Microbiomes: A system for studying natural complexity in a changing world - Kelly Speer
 
When asked what was the biggest challenge for managing these types of collections - physical storage (15-20 swabs per bat).

However she also mentioned linking of data, at which Arctos excels...

This was a great discussion about how pathogens and microbiomes interact with other biological entities. I wish these collections were already in Arctos.

## Innovations in Collections - Focused Education and Outreach - Shyla Davidson

A card trading game was demonstrated - an Arctos card trading game would be super cool.

Recommends using social media to interact with educators

## iDigBio Digitization Academy - Kalima Jacymec

 - Looking at creating new offerings and potential credentialling
 - There are currently 12 hour and 2 hour courses

**This made me think we need a regular Arctos training schedule in addition to office hours**

## On the Problems of digitizing indigenous knowledge - Jorge Soberon

@ArctosDB/diversity-and-inclusion should see this presentation if I can get a link to the recording, I will add it later.

The presentation centered on the difficulties and pitfalls of adding indigenous knowledge to a database.

 - WHICH indigenous knowledge?
 - WHO can speak for the indigenous community?
 - Modern science assumes a single correct knowledge but this is not the case in traditional societies
 - See the Convention on Biological Diversity Article 8(j)
 >  Subject to its national legislation, respect, preserve and maintain knowledge, innovations and practices of indigenous and local communities embodying traditional lifestyles relevant for the conservation and sustainable use of biological diversity and promote their wider application with the approval and involvement of the holders of such knowledge, innovations and practices and encourage the equitable sharing of the benefits arising from the utilization of such knowledge, innovations and practices; 

## Establishing the Digitization Coordination Network (DigiCoordNet) - David Jennings

I missed part of this because the concurrent schedule was not kept.

Some institutions have managed to hire a coordinator (the usual suspects) and this community would like to advocate for such a position, but is already relying on volunteers to get anything done. They have a Slack channel and a Github, but I was not able to access either of them.

## Exploring the AI kaleidoscope with VoucherVision: a spectrum of techniques for label transcription - William Weaver

See tools in resources above

## Developing an Oriented Object Detection Model for Museum Image Datasets - Bahadir Altintas

This was mostly over my head and I missed part due to scheduling probelms

## M-PABI - A database model for the Extended Specimens of a Pathogen Biorepository - Summer Menarelli

Missed due to schedule conflicts - want to see the recording!

## Trait Data from Herbarium Specimens: What is Needed? - Mason Heberling, Will Weaver and Rob Guralnick

 - Inclusive Herbaria include much more than just specimens
 - Standards are needed for trait data (TDWG?)
 - See links in resources above
 - Herbarium duplicates collect markers specific to their storage locations (that's pretty cool....)
 - How can tools used across the community be sustained?
 - Who is responsible for trait data and where should it be stored?

## My thoughts on Day 1

- Everyone is under-resourced
- AI is nice but people need jobs and are still better at most things?
- STOP asking people in this community to volunteer
- I'm ready to start working on the Arctos (digital) trading card game
- Why do people thing Social media is "low effort"? It takes a TON of work!
- Some standard monthly Arctos training would be a good idea
- Indigenous knowledge needs to be direct from the indigenous community, but who is the person that can give permission?

**Comment by Jegelewicz** [comment]
# Day 2 - May 30, 2024

## Interesting Resources

 - IsItABarrel: https://isitabarrel.ku.edu/ 
 - MorphoSource: https://www.morphosource.org/
 - Specify Collections Consortium: https://www.specifysoftware.org/
 - Lyrasis: https://www.lyrasis.org/Pages/Main.aspx
 - NobleReach Foundation: https://noblereachfoundation.org/
 - LTER Network: https://lternet.edu/
 - Coretrustseal: https://www.coretrustseal.org/
 - National Phenology Network: https://usanpn.org/
 - https://www.fieldmuseum.org/about/press/how-natural-history-museums-can-help-prevent-next-pandemic
 - The Bat Literature Project: https://bat-literature.github.io/
 - Environmental Data Science Innovation & Inclusion Lab (ESIIL): https://esiil.org/
 - Group notes from [Digital Collections Data and Tracking Disease Workshop](https://docs.google.com/document/d/1ml56nlXZ3E_kmsf0ksTJsXEtAKOhsRN3TjnABGbgusU/edit)

## Session Notes

## Outer membrane protein identification and AI generation - Joanna Slusky

- No versioning of data (OOF)
- This talk was VERY technical about the identification of outer cell membrane protein identification. Super cool, but over my head! However, it did end with the observation that the data needs curation.

## Planning for Sustainability - David Jennings

- Sustainability is at the center of Economic, Social and Environmental concerns
- Buzzwords: maintain, support, endure, continuity
- The path to sustainability is PLANNING
- But planning is hard because we don't do well at thinking long term
- "Linear timelines invite entropy"
- we have to embrace future uncertainty
- "It is not possible to sustain everything, everywhere, forever"
- iDigBio offers an 8 week strategic planning for collections course twice per year

## Morphosource: Sustaining 3D Data Access - Doug Boyer and Julie Winchester

- have funding through 2029
    - 4 FTE approx $400K annually
    - 107TB of storage approx $18.5K annually
 - expected maintenance expense
    - 2.5 FTE approx $350K annually
    - storage will only increase expected increases of $2.5K annually
 - it is projected that by 2040 annual budget will be about $500K
 - They have determined that they cannot require annual payments from contributors
 - Transfers of leadership/governance must be considered
 - What happens to the data if MorphoSource dies?
 - **Distributing management and infrastructure improves sustainability**
 - Looking at the following:
    - A volunteer consortium where members pay for a role in steering development
    - Charging deposit fees
    - Using tech to reduce operation costs (better storage management)
    - Scaling with internal infrastructure
    - Governance transition

## Sustaining Access to Biodiversity Occurrence Data - Jose Fortes

- 'Thinking thru the Unthinkable'
- **requires reliable, scalable, persistent, well-maintained, ever-evolving cyber infrastructure**

## Open Source Software, Stakeholders and $ustainability - Jim Beach

- To determine revenue and business model options they used Lyrasis
- They operat under an Industrial Affiliation Program at KU (available at most universities)
- Current membership
   - 151 institutions
   - 368 collections
 - **Annual revenue $500K** 
 - 6 full time FTE and 9 Students
 - They seek to add value at an institutional level because the institutions pay, not the collections
 - They ask how a CMS improves the sustainability of the physical collection

## Accelerating biotech development and translation: NSF and NobleReach EmergePilot - Charlene Marini

- NobleReach is meant to create sustainability through commercial translation
- Diversify funding
  - Grants
  - Philanthropy
  - Revenue
- **How have we never heard of this? It is what we needed at the beginning of the SABI and we should have been steered toward it....**

## Sustaining the future of science with open data at EDI - Greg Maurer

- The Environmental Data Initiative (EDI) is an LTER data repository
- Has Coretrustseal certification
- Also part of NobleReach to determine salable services
- Present **"Harmonized" or analysis-ready data products**
- **"Federal FAIR requirements are an unfunded mandate"**

## Adventure in Sustaining the USA National Phenology Network - Theresa Crimmins

- Funding goes to new and shiny things
- They have a SABI
- Are also a NobleReach project

## Balancing mission & opportunity for iDigBio sustainability - Dunrie Grieling

- Another NobleReach project!
- OOF on the mission of iDigBio - Where is my cloud computing platform?

## Digital Collections Data and Tracking Disease Workshop
## A harmonized taxonomic resource is critical for accurately interpreting host-pathogen interactions - Kendra Phelps

- BTA tool
- Asked to contribute to the GBIF backbone

## Preserve a Voucher Specimen! - Colby Thomspon

- What can NHCs do to be part of the solution?
- (My thoughts - STOP using the word voucher because it is meaningless!)

## Virome Composition in Fresh Bat Guano, Frozen Bats and Fluid-preserved bat tissues - Verit Mathis

- frozen had best results
- we need more economical methods

## Exploring bat coronaviruses using the FMNH cryo collection - Molly McDonough, Adam Ferguson

- I lost power and missed most of this, but it seems like the Field did a great job of including students during the pandemic on research about coronavirus

## Targeted sequencing of pathogen DNA from museum specimens - Neal Platt, Texas Biomedical Research Institute

## Preventing the next viral pandemic via elucidation of mammalian collections - Rick White

- using protein sequences instead of RNA/DNA relieves the need for LN2

**Reply by jhpoelen** [reply]
@Jegelewicz thanks for openly sharing your notes on Digital Data 8 . I didn't attend the conference, and your notes helped me to figure out what happened.

I noted the talk by Town Peterson (a keynote?) on "blockchain."

> Managing Biodiversity Data in the Blockchain: Full Tracking of All Data Transactions Through Time in > Managing Systematic Collections - Town Peterson
> Current Biodiversity Data Challenges
>
> Records are dynamic
> Data exists as a 'single final version' without retention of previous versions
> No changes tracking (not true for some things in Arctos)
> No user-based tracking (not true for some things in Arctos)

I like to idea of trying out new ideas, and I prefer to show a working system to do so. For this topic of keeping track of biodiversity data, I would have talked about an ongoing effort to keep track of *all* (yes, all) biodiversity datasets registered with GBIF and iDigBio via monthly snapshots. With this, we now have an openly accessible archive of versioned copies of DwC-A and EML records from period 2018/2024 (now). This corpus, among others, is mobile and is kept at BioKIC, internet archive, a server in Germany, and on hard disks under my pillow.  A similar approach was taken to compile a corpus of 20k works as part of the [https://taxodros.github.io](https://taxodros.github.io) and [https://bat-literature.github.io](https://bat-literature.github.io) projects. 

You can make your own copy of these corpora too . . . For instance, if you have time bandwidth and space to transfer and keep (~5TB) the largest openly available of collection of biodiversity data records that I know of, follow the steps below - 

```
# get latest version reference
latest_version=preston head --remote https://linker.bio

preston clone\
 --anchor "${latest_version}"\
 https://linker.bio
```

with latest_version being ```hash://sha256/5ce64ad8b2bd4d209eb1fc638c246defa5ca70555cfb49f4fb2134e2920439f6``` at time of writing. 

The following two papers discuss the approach and the benefits of keeping track of specific versions of digital data without compromising their ability to be updated - 

Elliott M.J., Poelen J.H., Fortes J.A.B. (2020). Toward Reliable Biodiversity Dataset References. Ecological Informatics. https://doi.org/10.1016/j.ecoinf.2020.101132 [hash://sha256/136c3c1808bcf463bb04b11622bb2e7b5fba28f5be1fc258c5ea55b3b84f482c](https://linker.bio/hash://sha256/136c3c1808bcf463bb04b11622bb2e7b5fba28f5be1fc258c5ea55b3b84f482c)

Elliott M.J., Poelen, J.H. & Fortes, J.A.B. (2023) Signing data citations enables data verification and citation persistence. Sci Data. https://doi.org/10.1038/s41597-023-02230-y [hash://sha256/f849c870565f608899f183ca261365dce9c9f1c5441b1c779e0db49df9c2a19d](https://linker.bio/hash://sha256/f849c870565f608899f183ca261365dce9c9f1c5441b1c779e0db49df9c2a19d)

Also, see text on [https://linker.bio](https://linker.bio) for an attempt to a more "popular" description on the topic. 

All these use so-called "blockchain" technology. Note that git, the versioning system (aka content tracker) underlying github, gitlab, is also a kind of "blockchain" technology. See the method section of Elliott et al. 2023 for a practical demonstration of securely chaining blocks of data via cryptographic hashes (aka content ids).

I am curious to see the recording of the talk, as I wonder whether Town Peterson et al. made an effort to cite prior work on the topic.

-jorrit
https://jhpoelen.nl

**Comment by Jegelewicz** [comment]
# Day Three - May 31, 2024

## Interesting Resources

- Voyant Tools: https://voyant-tools.org/
- Sublime Text Editing: https://www.sublimetext.com/
- Komodo Text Editing: https://www.activestate.com/products/komodo-edit/
- Visual Studio Code Editing: https://code.visualstudio.com/
- iDigBio Data Quality Toolkit: https://www.idigbio.org/wiki/index.php/Data_Quality_Toolkit_2024
- GeoNames: https://www.geonames.org/export/codes.html

## Opportunities to Synthesize Data Provided by Long Term Research - Allen Moore

- LTER and ESIIL
- None of the discussed programs support museums?
- What I heard
  - There are no plans for sustainability of the data
  - There are no data standards in use

## Data Cleaning for maximum impact (Workshop)

- What is data quality?
  - fitness for use?
  - fidelity?
  - consistency?
  - IBM: "Data quality measures how well a dataset meets criteria for accuracy, completeness, validity, consistency, uniqueness, timeliness and fitness for purpose, and it is critical to all data governance initiatives within an organization." https://www.ibm.com/topics/data-quality

## Exploring Data Management and Curation in TaxonWorks - Debbie Paul and Tommy McElrath

- They include a DarwinCore importer, seems like a great idea!
- Query results can be passed to other queries
- The include a DarwinCore search results option, also seems like a great idea!
- We could get a TaxonWorks sandbox....
- They meet weekly on Wednesdays

## Arctos Data Quality Tools - Teresa Mayfield-Meyer

- Search Arctos https://arctos.database.museum/
- Arctos Handbook https://handbook.arctosdb.org/
- About Arctos https://arctosdb.org/ 
- Arctos Github https://github.com/ArctosDB/arctos/issues
- Data Quality topics covered
  - GADM and IHO Seas for higher geography
  - Code Tables
  - Agents
  - Taxonomy
  - Duplicate catalog numbers not allowed
  - All search options
  - TURD
- Answered a question about searching pathogen relationships

## Data Quality Workshop Wrap-up: what have we learned?

- Train your staff/volunteers/anyone who works with the data
- Set up workflows and protocols that are documented and shared
- Add resources to a shared document (the data quality toolkit)
- Pay it forward!
- Make corrections in batches when possible (but don’t make bigger messes….!

**Comment by AJLinn** [comment]
From your Day 1 notes:

> On the Problems of digitizing indigenous knowledge - Jorge Soberon

In my view it's all about informed consent, co-production of knowledge, and attribution (if wanted by the Indigenous partners). I'm working on submitting a proposal to NSF for this summer that will look at some of these questions and we're budgeting funding for Arctos to help us build some levels of protection and partnership for Iñupiaq language and knowledge. Encumbrances (or curatorial remarks, if that's where we land) and various levels of user access will help us with that.

> Indigenous knowledge needs to be direct from the indigenous community, but who is the person that can give permission?

In Alaska we work thru the IRA Tribal Councils to gain permission. Many of them have their own IRB-type departments or staff who deal with working with researcher or have protocols for this. For example, check out Kawerak's resources: https://kawerak.org/natural-resources/research/# and https://kawerak.org/natural-resources/social-science/.

**Comment by AJLinn** [comment]
> A card trading game was demonstrated - an Arctos card trading game would be super cool.

Here's one from UAMN for reference: https://www.uaf.edu/museum/exhibits/exhibition-resources/collect-alaska-table-top-/index.php
